application/src/bll/reducers/paymentCardReducer.test.ts

```diff
@@ -0,0 +1,44 @@
+import { v1 } from 'uuid';
+import {
+  paymentCardReducer,
+  setCustomerData,
+  saveDataOrders
+} from './paymentCardReducer';
+
+describe('payment card reducer', () => {
+  test('set customer data', () => {
```

Comment:
What do we check in this test? Please update name

`should show default customer data`,
`should set customer data`
etc.

Reply:
i fixed everywhere